styles/stylesheet.css

```diff
@@ -0,0 +1,251 @@
+* {
+	box-sizing: border-box;
+}
+
+body {
+	background: white;
+	height: 100%;
+	padding: 1rem;
+}
+
+.header {
+	background-repeat: no-repeat;
+	max-width: 100%;
+	height: 400px;
+	background-image: url("../images/header-image.jpg");
+	background-size: cover;
+}
+
+.header-logo {
+	max-width: 100%;
+	height: auto;
+	float: left;
+	padding: 3rem;
+}
+
+.header-logo img {
+	float: left;
+}
+
+.slogan {
+	font-family: "Comic Sans MS", cursive, sans-serif;
+	font-size: 1.5rem;
+	color: #1B8F1B
+}
+
+.shoppingcart {
+	float: right;
+	padding: 0.5rem;
+}
+
+.shoppingcart img {
```

Comment:
You'll want to evaluate how you're sizing your shoppingcart image. It appears to be extremely large on all screen sizes.